### PR TITLE
(DOCSP-28190) Andrew's RN SDK docs feedback - Round 3: Dictionaries

### PR DESCRIPTION
**Pull request by krollins-mdb**
### Jira

- https://jira.mongodb.org/browse/DOCSP-28190

### Staged Changes

- [Dictionaries](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-28190-round3/sdk/react-native/realm-database/schemas/dictionaries/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by MongoCaleb — APPROVED** [review]
Staging link is borked, but LGTM

**Comment by krollins-mdb** [comment]
Agh, I knew I forgot something. Fixed the staging link. Thanks, Caleb!

File: source/sdk/react-native/realm-database/schemas/dictionaries.txt [hunk 1]
```diff
@@ -12,20 +12,27 @@ Dictionaries - React Native SDK
...
-To define a dictionary of mixed values in your schema, set the data type
-of your field to an empty object, ``"{}"``. Alternatively, add the data type before the brackets to create a
-dictionary with values of a specific type. For instance, ``"int{}"`` to specify that dictionary values must be
-integers or ``"string{}"`` to specify that dictionary values must be strings.
+To define a dictionary of mixed values in a :ref:`Realm object model
+<react-native-define-a-realm-object-model>`, set the data type
+of your field to an empty object, ``"{}"``. Alternatively, add the data type
+before the brackets to create a dictionary with values of a specific type. For
+instance, ``"int{}"`` to specify that dictionary values must be integers or
+``"string{}"`` to specify that dictionary values must be strings.
```
Comment (takameyer) [inline source/sdk/react-native/realm-database/schemas/dictionaries.txt:35]: It's also possible to define a dictionary with the `object` syntax, which would be required if you wanted to have a dictionary that included custom objects.  For example:
```
properties: {
  children: {type: "dictionary", objectType: "Person"}
}
```

**Review by takameyer — COMMENTED** [review]
Just one suggestion.  Otherwise LGTM! :shipit:

**Review by takameyer — APPROVED** [review]

File: source/sdk/react-native/realm-database/schemas/dictionaries.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/sdk/react-native/realm-database/schemas/dictionaries.txt]: Thanks, Andrew! I've clarified this in the example.

**Pull request merged**